code/__HELPERS/path.dm

```diff
@@ -130,7 +133,7 @@
  * search() is the proc you call to kick off and handle the actual pathfinding, and kills the pathfind datum instance when it's done.
  *
  * If a valid path was found, it's returned as a list. If invalid or cross-z-level params are entered, returns FALSE.
- * If no valid path is found, returns an empty list, which is important for simple bot mobs who rely on an empty list meaning no path found.
+ * If no valid path is found, returns FALSE or null, which [/proc/get_path_to] translates to an empty list (notable for simple bots, who need empty lists)
```

Comment:
What does returning `FALSE` mean *specifically*, any why doesn't it either always return `FALSE` or `null`? Is there an important distinction to the consumer?

Reply:
You're right, I'm gonna make it consistent since there is no difference really